tests/dragonfly/replication_test.py

```diff
@@ -1971,23 +1971,27 @@ async def test_replicaof_reject_on_load(df_factory, df_seeder_factory):
     df_factory.start_all([master, replica])
 
     c_replica = replica.client()
-    await c_replica.execute_command(f"DEBUG POPULATE 8000000")
+    await c_replica.execute_command(f"DEBUG POPULATE 4000000")
 
     replica.stop()
     replica.start()
     c_replica = replica.client()
+
+    @assert_eventually
```

Comment:
why do you use the assert_eventually here? shouldnt we see the loading state at the first time we run the info persistence?